airbyte-integrations/connectors/source-aha/source_aha/source.py

```diff
@@ -16,3 +19,9 @@
 class SourceAha(YamlDeclarativeSource):
     def __init__(self):
         super().__init__(**{"path_to_yaml": "manifest.yaml"})
+
+    def check_connection(self, logger, config) -> Tuple[bool, any]:
+        response = requests.head(urljoin(config['url'], f"/api/v1/products"), headers={"Authorization": f"Bearer {config['api_key']}"})
+        if response.status_code == 200:
+            return True, None
+        return False, response.text
```

Comment:
Aha Source already uses the `check_connection` from the low-code CDK.

https://github.com/airbytehq/airbyte/blob/a6990bc1958c28fcc4718676f8fc5f9710cdb108/airbyte-integrations/connectors/source-aha/source_aha/manifest.yaml#L103-L109